ui/pages/reservation-unit/[id].tsx

```diff
@@ -1049,140 +1043,105 @@ const ReservationUnit = ({
+                      />
                     </p>
                   )}
                   {reservationUnit.minReservationDuration &&
                     reservationUnit.maxReservationDuration && (
                       <p>
-                        <Trans i18nKey="reservationUnit:reservationInfo3">
-                          Varauksen keston tulee olla välillä{" "}
-                          <strong>
-                            {{
-                              minReservationDuration: formatSecondDuration(
-                                reservationUnit.minReservationDuration,
-                                false
-                              ),
-                            }}
-                          </strong>{" "}
-                          ja{" "}
-                          <strong>
-                            {{
-                              maxReservationDuration: formatSecondDuration(
-                                reservationUnit.maxReservationDuration,
-                                false
-                              ),
-                            }}
-                          </strong>
-                          .
-                        </Trans>
+                        <Trans
+                          defaults="Varauksen keston tulee olla välillä <bold>{{minReservationDuration}}</bold> ja <bold>{{maxReservationDuration}}</bold>."
+                          values={{
+                            minReservationDuration: formatSecondDuration(
+                              reservationUnit.minReservationDuration,
+                              false
+                            ),
+                            maxReservationDuration: formatSecondDuration(
+                              reservationUnit.maxReservationDuration,
+                              false
+                            ),
+                          }}
+                          components={{ bold: <strong /> }}
+                        />
                       </p>
                     )}
                   {reservationUnit.maxReservationsPerUser && (
                     <p>
                       <Trans
-                        i18nKey="reservationUnit:reservationInfo4"
                         count={reservationUnit.maxReservationsPerUser}
-                      >
-                        Sinulla voi olla samanaikaisesti{" "}
-                        <strong>
-                          enintään{" "}
-                          {{ count: reservationUnit.maxReservationsPerUser }}{" "}
-                          varausta
-                        </strong>
-                        .
-                      </Trans>
+                        defaults="Sinulla voi olla samanaikaisesti <bold>enintään {{count}} varausta</bold>."
+                        values={{
+                          count: reservationUnit.maxReservationsPerUser,
+                        }}
+                        components={{ bold: <strong /> }}
+                      />
```

Comment:
Missing i18nKey. Need to refactor the translation tags.
See https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0